### DISCUSSION TITLE
Credential Types

### DISCUSSION THREAD

**Discussion by simibac** [decentralized-identity/veramo#390] [Q&A] [answered]
When creating VC with the CLI, the third question is about the Credential Type. The default value is `VerifiableCredential,Profile` and works fine. However, the `@context` that is generated only references to `https://www.w3.org/2018/credentials/v1` which does not contain a type of `Profile`. IMO it should also contain a second context that defines the `Profile`. Is this still work in progress or am I wrong about this?

Also is there a way to manually add a second context during the VC generation process?

**Comment by mirceanis** [comment, marked as answer]
Thank you for flagging this behavior. It's a bug, or rather the lack of a feature.

There is a plan to add some more flexibility to the CLI commands, but it is lower down in the priority list (for now).
Many commands have an interactive aspect and the plan is to enable the user to specify all the parameters in advance so that the CLI can be used without human presence.
In this case, it would be great if you could pipe a JSON file with the complete payload (including custom contexts) to create the verifiable credential.

If you have a suggestion about how this could be improved, and especially if you want to contribute a fix, please open an issue and/or PR.
